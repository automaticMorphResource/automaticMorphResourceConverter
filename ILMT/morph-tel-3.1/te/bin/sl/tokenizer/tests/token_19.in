కంగారూ దీవి... ఇది ఏషియా ఫసిఫిక్ ప్రాంతంలో ఉండే అన్ని దీవుల్లోకెల్లా అందమైనదిగా పేరుగాంచింది. 
