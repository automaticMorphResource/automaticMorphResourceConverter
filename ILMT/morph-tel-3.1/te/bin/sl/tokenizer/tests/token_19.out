<Sentence id="1">
1	కంగారూ	unk
2	దీవి...	unk
3	ఇది	unk
4	ఏషియా	unk
5	ఫసిఫిక్	unk
6	ప్రాంతంలో	unk
7	ఉండే	unk
8	అన్ని	unk
9	దీవుల్లోకెల్లా	unk
10	అందమైనదిగా	unk
11	పేరుగాంచింది	unk
12	.	unk
</Sentence>

